content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -7,6 +7,8 @@ weight: 20
 min-kubernetes-server-version: 1.19
```

Comment:
Remember to
```suggestion
min-kubernetes-server-version: 1.20
```
:wink:

Reply:
added `"{{< skew latestVersion >}}"` here, instead of the literal version.